src/config.rs

```diff
@@ -147,6 +152,10 @@ impl S3Config {
         }?)
     }
 
+    fn load_key_expr(config: &StorageConfig) -> ZResult<OwnedKeyExpr> {
+        Ok(config.key_expr.to_owned())
+    }
+
```

Comment:
For the sake of readability I had thought, since we have multiple other 'load' functions called upon construction of the S3Config struct. But yes, maybe that's not enough justification for this one line function.

Reply:
Please remove it.